docs/ref/dotq.md

```diff
@@ -1324,6 +1324,7 @@ select … [by date,…] from … where [date …]
 
     For Linux see `vm.max_map_count`.
 
+Since 4.1t 2024.01.11 parallelized over tables and partitions with [peach](each.md) when kdb+ running with secondary threads.
```

Comment:
```suggestion
Since 4.1t 2024.01.11 parallelized over tables and partitions with [Peach](each.md) when kdb+ is running with secondary threads.
```

Reply:
Not sure if peach is the name of a thing. If it is, it needs a capital letter throughout.

Reply:
peach should be lowercase (as typed in cmd)